starknet-rpc-test/add_declare_transaction.rs

```diff
@@ -145,13 +147,14 @@ async fn works_with_storage_change(madara: &ThreadSafeMadaraClient) -> Result<()
         }
         _ => panic!("Expected declare transaction result"),
     }
-
-    assert!(rpc.get_class(BlockId::Number(block_number), expected_class_hash).await.is_ok());
-
     // included in block
     let included_txs = rpc.get_block_transaction_count(BlockId::Number(block_number)).await?;
     assert_eq!(included_txs, 1);
 
+    // Wait a bit to let the client sync the block
+    sleep(Duration::from_secs(3)).await;
```

Comment:
realistically how long does it take ?

Reply:
its not blocking for anything else than `get_class` right ?

Reply:
I'm not sure. Probably less than 1sec.

Indeed, it's only an issue for `get_class` and `get_class_at`